### PR TITLE
Add a short blog post + slids of the talk at the Python Language Summit

### PR DESCRIPTION
**Pull request by antocuni**
preview: https://deploy-preview-13--festive-wing-250462.netlify.app/

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -247,7 +247,10 @@
 # One or more folders containing files to be copied as-is into the output.
 # The format is a dictionary of {source: relative destination}.
 # Default is:
-# FILES_FOLDERS = {'files': ''}
+FILES_FOLDERS = {
+    'files/talks': 'talks',
+    #'files': ''
```
Comment (hodgestar) [inline conf.py:252]: The comment above seems to suggest that `'files': ''` should do exactly what you want if you create a `talks` folder within files?

**Review by hodgestar — APPROVED** [review]
One small suggestion to simplify the files configuration, but otherwise looks good to me.

File: conf.py (hunk 1 above)
Comment (antocuni) [reply conf.py:252]: uhm, maybe. Let me try :)

Comment (antocuni) [reply conf.py]: indeed, it works. See d27bd77, thanks

**Comment by timfel** [comment]
Looks good!

**Review by hodgestar — APPROVED** [review]

**Pull request merged**